samples/drawing/drawing.cpp

```diff
@@ -2142,17 +2157,17 @@ MyFrame::MyFrame(const wxString& title, const wxPoint& pos, const wxSize& size)
 
     wxMenu *menuFile = new wxMenu;
 #if wxUSE_GRAPHICS_CONTEXT
-    menuFile->AppendCheckItem(File_GC_Default, "Use default wx&GraphicContext\tCtrl-Y");
-    m_menuItemUseDC = menuFile->AppendRadioItem(File_DC, "Use wx&DC\tShift-Ctrl-Y");
+    menuFile->AppendCheckItem(File_GC_Default, "Use default wx&GraphicContext\tCtrl-1");
+    m_menuItemUseDC = menuFile->AppendRadioItem(File_DC, "Use wx&DC\tCtrl-2");
```

Comment:
Really a nitpick, but should this be `Ctrl-0` perhaps? Using DC is different from all the other ones...

Reply:
Done, also removed `Ctrl`.